src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -407,11 +407,19 @@ private Point getBlockTransclusionLocation(int index) {
 		JavascriptExecutor js = (JavascriptExecutor) driver;
 		Object templateBounding = js.executeScript(VEContent.BOUNDING_SCRIPT, blockTransclusionString, index);
 		Map<String, String> mapBounding = (Map) templateBounding;
-		int tempLeft = Integer.parseInt(String.valueOf(mapBounding.get("left")));
```

Comment:
<code>
int tempLeft = Integer.parseInt(String.valueOf(mapBounding.get("left")));
</code>
to 
<code>
int tempLeft = new BigDecimal(String.valueOf(mapBounding.get("left"))).intValue();
</code>
Does it solve the issue?